instrumentation/logger/Gemfile

```
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+source 'https://rubygems.org'
+
+# DO NOT ADD DEPENDENCIES HERE!
+# Please declare a minimum development dependency in the gemspec,
+# then target specific versions in the Appraisals file.
+
+gemspec
+
+group :test do
+  gem 'opentelemetry-instrumentation-base', path: '../base'
+  gem 'opentelemetry-api', path: '/Users/kreopelle/dev/opentelemetry-ruby/api'
+  gem 'opentelemetry-logs-api', path: '/Users/kreopelle/dev/opentelemetry-ruby/logs_api'
+  gem 'opentelemetry-logs-sdk', path: '/Users/kreopelle/dev/opentelemetry-ruby/logs_sdk'
+  gem 'opentelemetry-sdk', path: '/Users/kreopelle/dev/opentelemetry-ruby/sdk'
```

Whoops, didn't mean to commit my local paths. This will install things with the latest SDK branch, but Appraisal is having trouble installing gems via git, so this might not be the best solution:
```suggestion
  gem 'opentelemetry-api', git: 'https://github.com/kaylareopelle/opentelemetry-ruby', branch: 'log-record-processor3', glob: 'api/*.gemspec'
  gem 'opentelemetry-logs-api', git: 'https://github.com/kaylareopelle/opentelemetry-ruby', branch: 'log-record-processor3', glob: 'logs_api/*.gemspec'
  gem 'opentelemetry-logs-sdk', git: 'https://github.com/kaylareopelle/opentelemetry-ruby', branch: 'log-record-processor3', glob: 'logs_sdk/*.gemspec'
  gem 'opentelemetry-sdk', git: 'https://github.com/kaylareopelle/opentelemetry-ruby', branch: 'log-record-processor3', glob: 'sdk/*.gemspec'
```